src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/TransactionResponse.cs

```diff
@@ -7,7 +7,10 @@ internal class TransactionResponse : Identifiable, ITransactionResponse
         public TransactionResponse(Uri id, TransactionDto transaction)
             : base(id)
         {
-            Transaction = transaction.Map();
+            if(transaction != null)
+            {
+                Transaction = transaction.Map();
+            }

```

Comment:
`null` propagation FTW?

```suggestion
            Transaction = transaction?.Map();
```